.scalafmt.conf

```diff
@@ -75,3 +75,19 @@ project.excludeFilters = [
   "scripts/authors.scala"
 ]
 project.layout         = StandardConvention
+
+fileOverride {
+  "glob:**/scala-3/**" {
+    runner.dialect = scala3
+    rewrite.scala3.convertToNewSyntax = false
+  }
+}
+
+//rewrite.scala3.convertToNewSyntax = false
+//runner {
+//  dialectOverride {
+//    allowAsForImportRename = false
+//    allowStarWildcardImport = false
+//    allowPostfixStarVarargSplices = false
+//  }
+//}
```

Comment:
nit: newline

Reply:
How do you think we migrate to the `?`.

Reply:
I meant that the file is missing a new line at the end, thats all